go-controller/pkg/ovn/namespace.go

```diff
@@ -3,10 +3,42 @@ package ovn
 import (
 	"github.com/Sirupsen/logrus"
 	kapi "k8s.io/api/core/v1"
+	"os/exec"
+	"strings"
 	"sync"
 	"time"
 )
 
+func (oc *Controller) syncNamespaces(namespaces []interface{}) {
+	expectedNs := make(map[string]bool)
+	for _, nsInterface := range namespaces {
+		ns, ok := nsInterface.(*kapi.Namespace)
+		if !ok {
+			logrus.Errorf("Spurious object in syncNamespaces: %v", nsInterface)
+			continue
+		}
+		expectedNs[ns.Name] = true
+	}
+	// get the list of address sets from OVN
+	output, err := exec.Command(OvnNbctl, "--data=bare", "--no-heading",
+		"--columns=external_ids", "find", "address_set").Output()
+	if err != nil {
+		logrus.Errorf("Error in obtaining list of address sets from OVN: %v",
+			err)
+		return
+	}
+	for _, addrSet := range strings.Fields(string(output)) {
+		addrSetName := addrSet[5:]
+		names := strings.Split(addrSetName, ".")
+		if len(names) == 1 && names[0] != "kube-public" &&
+			names[0] != "kube-system" && names[0] != "default" &&
```

Comment:
Should probably have a discrete function called "isKubeSystemNamespace()" that returns a bool for this.

Reply:
I found that we actually don't need this. Removed from latest ps.